network-api/networkapi/utility/templatetags/active_nav.py

```diff
@@ -0,0 +1,15 @@
+from django import template
```

Comment:
copied from https://maorleger.wordpress.com/2015/11/20/active-page-styling-for-bootstrap-navbar-and-django/

should I leave a comment here and cite that URL?

Reply:
Probably useful for future reference.